src/pages/invest.tsx

```diff
@@ -0,0 +1,16 @@
+import Layout from "@components/Layout";
+import React, { ReactElement } from "react";
+
+function Invest() {
```

Comment:
rename to InvestPage

Reply:
because in my head the component is the invest page but it's also not a hill to die on for me